pom.xml

```diff
@@ -75,8 +75,8 @@
 
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <cdap.version>6.8.0</cdap.version>
-    <cdap.plugin.version>2.10.0</cdap.plugin.version>
+    <cdap.version>6.8.1-SNAPSHOT</cdap.version>
+    <cdap.plugin.version>2.10.1-SNAPSHOT</cdap.plugin.version>
```

Comment:
dependencies should not be upgraded unless they are needed
like we add an API that the plugin needs to use.